proto/frequenz/api/common/v1/pagination.proto

```diff
@@ -0,0 +1,34 @@
+// Parameters for pagination.
+//
+// Copyright 2023 Frequenz Energy-as-a-Service GmbH
+//
+// Licensed under the MIT License (the "License");
+// you may not use this file except in compliance with the License.
+
+syntax = "proto3";
+
+package frequenz.api.common.v1.pagination;
+
+// A message defining parameters for paginating list requests.
+message PaginationParams {
+  // [Optional]
+  // The maximum number of results to be returned per request.
+  // Defaults to 50 if omitted.
+  uint32 page_size = 1;
```

Comment:
please add `optional` as a prefix. I missed that.

Reply:
The word `Optional` in the docs looks incorrect since the field itself is not optional.

We should make the `page_size` field optional, right? If not specified, then it could default to service's page size (we'd need to mention this in the docs).

Reply:
> We should make the page_size field optional, right? If not specified, then it could default to service's page size (we'd need to mention this in the docs).

true, makes sense.

Reply:
```suggestion
  optional uint32 page_size = 1;
```